### ISSUE TITLE
E127: bugs?

### ISSUE THREAD

**Issue by StanTraykov** [PyCQA/pycodestyle#758]
New to Python, so sorry if I missed something, but E127 gives me trouble on two (probably related) occasions:

### continuing a statement after backslash
```
assert self.foobar in ['bar', 'baz', 'befoozle'], \
        'Invalid foobar: {0}'.format(self.foobar)
if demo:
    normal_indent = 4 * space
```
Here, the continuation of ``assert`` is double-indented, in line with the usual practice of distinguishing indents due to breaks from those that follow program logic (as also discussed in PEP8 with slightly different examples). Is there a good reason ``pycodestyle`` doesn't like this (E127)?

### aligning operators after backslash
```
self.foozle_timeout = max(self.foozles_left, 1) * foozle_time \
                      + self.foozle_tolerance
```
I understand that PEP8 recommends using parentheses/brackets/braces when possible, eliminating the need of a backslash. But still, it seems wrong that this is turned down (it follows the same operator alignment as in the PEP8 Knuth discussion, but without parentheses). If ``pycodestyle`` really wants to insist on using parentheses instead of a backslash here, it should do so explicitly, not by issuing E127.

EDIT: Maybe I don't strictly have a cause to complain in the second issue, as this type of "visual" indent is discussed in PEP8 only in the context of aligning with opening delimiters. But I am not certain it should be forbidden, and the first issue seems unaffected.

**Comment by plammens** [comment]
I think both are related to #569

**Comment by asottile** [comment]
Seems fixed on master, I believe this was resolved by #912
